dotnet-test-samples/hexagonal-architecture/template.yaml

```diff
@@ -0,0 +1,49 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+
+Globals:
+  Function:
+    MemorySize: 1024
+    Architectures: [arm64]
+    Runtime: dotnet6
+    Timeout: 30
+    Tracing: Active
+
+Resources:
+  StocksConverterFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: ./src/GetStock/bin/Release/net6.0
```

Comment:
What compiles the application code into this folder? Typically, I have seen the CodeUri pointing to the folder containing csproj and SAM runs the build

Reply:
You're right, fixed